### DISCUSSION TITLE
Red Warning Icon using Github Pages Server Type

### DISCUSSION THREAD

**Discussion by carstenfeller** [GetPublii/Publii#1780] [Help] [answered]
Hi there,
I'm using the "old" Github Sync, because the new Git Repository gives me some 404 I can't resolve atm.

Please see screenshot.

![publii-github-success](https://github.com/user-attachments/assets/1ccbba1d-172b-420e-8e80-2451c8d85122)

Should I be worried about the red icon?

Sync is running pretty fine.

Thanks.
Carsten

**Comment by dziudek** [comment, marked as answer]
@carstenfeller - don't worry - it will be removed in v.0.46.2 planned for this week :)

**Reply by carstenfeller** [reply]
thanks @dziudek 

so it's fine to continue using the "old" integration?

the 404 with the "Git Repository" server type I can't fix, shows this message:

![gitrepository-404](https://github.com/user-attachments/assets/f220e90c-293f-4601-8cc8-922f1cbe7178)

nothing to see in the logs.

**Reply by carstenfeller** [reply]
Nevermind, got it sorted out :-)
Was me, messing up with login credentials... 😵‍💫
